src/pages/Home/index.tsx

```diff
@@ -79,28 +79,30 @@ const parseHashRate = (hashRate: string | undefined) => {
   return hashRate ? `${localeNumberString((Number(hashRate) * 1000).toFixed(), 10)} gps` : '- -'
 }
 
-const TableTitleDatas: TableTitleData[] = [
-  {
-    title: i18n.t('home.height'),
-    width: '14%',
-  },
-  {
-    title: i18n.t('home.transactions'),
-    width: '14%',
-  },
-  {
-    title: i18n.t('home.block_reward'),
-    width: '20%',
-  },
-  {
-    title: i18n.t('block.miner'),
-    width: '37%',
-  },
-  {
-    title: i18n.t('home.time'),
-    width: '15%',
-  },
-]
+const tableTitles = () => {
```

Comment:
Why use the function to return the constant.

Reply:
Because we should support English and Chinese switching, when I switch other language, all `i18n` elements should render again.

Use function instead of const to make render success.

Reply:
use the [hook](https://react.i18next.com/latest/usetranslation-hook) for dynamically switching

Reply:
I consider using hook, but `contexts -> language` change language through `i18n`.

If I use `useMemo`, the dependence will not include `language` directly because these const does not include `language`.

Reply:
```js
const tableTitles = useMemo(() => {
   // something which not include language
}, [language])
```

Reply:
I thinks the following code should work.
```javascript
const title = useMemo(() => {
  return [
    {
      title: t('home.height'),
      width: '14%',
    },
    {
      title: t('home.transactions'),
      width: '14%',
    },
    {
      title: t('home.block_reward'),
      width: '20%',
    },
    {
      title: t('block.miner'),
      width: '37%',
    },
    {
      title: t('home.time'),
      width: '15%',
    },
  ]
}, [t])
```

Reply:
OK, I will have a try.

Reply:
Fixed. 
https://github.com/nervosnetwork/ckb-explorer-frontend/pull/261/commits/3451de58ddcd9a1224e9d3ee58f926ceeebc7c77

https://github.com/nervosnetwork/ckb-explorer-frontend/pull/261/commits/7d4f4cf83196690cf9461e0c9d9a68b76eaacbc3